src/sentry/snuba/metrics/query.py

```diff
@@ -42,43 +65,152 @@ class MetricsQuery:
     granularity: Granularity
     where: Optional[ConditionGroup] = None  # TODO: Should restrict
     groupby: Optional[Sequence[Groupable]] = None
-    orderby: Optional[OrderBy] = None
+    orderby: Optional[Sequence[OrderBy]] = None
```

Comment:
As mentioned in my other comment, I would leave this type as-is. The metrics layer does not support multiple order bys, so why not reflect that in the type?

Reply:
Actually we will have to start supporting it for fields of the same entity for MEP
Do you have any strong opinions against that?

Reply:
I see, no strong opinions against that! In that case, ignore my comments :)